README.md

```diff
@@ -58,32 +58,33 @@ The Stripe Android SDK will configure your app's Proguard rules using [proguard-
 
-**Google Pay**: Stripe is fully compatible with [Google Pay](https://stripe.com/docs/google-pay).
+**SCA-Ready**: The SDK automatically performs native [3D Secure authentication](https://stripe.com/docs/payments/3d-secure) to comply with [Strong Customer Authentication](https://stripe.com/docs/strong-customer-authentication) regulation in Europe.
 
-**Stripe API**: We provide [low-level APIs](https://stripe.dev/stripe-android/stripe/com.stripe.android/-stripe/index.html) that correspond to objects and methods in the Stripe API. You can build your own entirely custom UI on top of this layer.
+**Native UI**: We provide native screens and elements to collect payment. For example, [PaymentSheet](https://stripe.com/payments/accept-a-payment?platform=android) is a prebuilt UI that combines all the steps required to pay - collecting payment details, billing details, and confirming the payment  - into a single sheet that displays on top of your app.
 
-**SCA-Ready**: The SDK automatically performs native [3D Secure authentication](https://stripe.com/docs/payments/3d-secure) to comply with [Strong Customer Authentication](https://stripe.com/docs/strong-customer-authentication) regulation in Europe.
+<img src="https://raw.githubusercontent.com/stripe/stripe-android/michelleb/mc-update-readme/assets/payment_sheet_complete.png"/>
 
-**Native UI**: We provide native screens and elements to collect payment and shipping details. For example, [CardInputWidget](https://stripe.dev/stripe-android/stripe/com.stripe.android.view/-card-input-widget/index.html) is a view that collects and validates card details. You can use these individually, or take all of the prebuilt UI in one flow by following the [Basic Integration guide](https://stripe.com/docs/mobile/android/basic).
+**Google Pay**: Stripe is fully compatible with [Google Pay](https://stripe.com/docs/google-pay).
+
+**Stripe API**: We provide [low-level APIs](https://stripe.dev/stripe-android/stripe/com.stripe.android/-stripe/index.html) that correspond to objects and methods in the Stripe API. You can build your own entirely custom UI on top of this layer.
 
-<img width="270" height="555" src="https://raw.githubusercontent.com/stripe/stripe-android/master/assets/card_input.gif"/>
 
 ## Getting Started
 
 Get started with our [📚 integration guides](https://stripe.com/docs/payments) and [example projects](#examples), or [📘 browse the SDK reference](https://stripe.dev/stripe-android/).
 
-The `Stripe` class is the entry-point to the Stripe SDK. It must be instantiated with a [Stripe publishable key](https://stripe.com/docs/keys).
+The `PaymentSheet` class is the entry-point to the pre-built UI. It must be instantiated with a [Stripe publishable key](https://stripe.com/docs/keys) and application context.
```

Comment:
Can you clarify what it means that it needs to be instantiated with a publishable key and application context? Do you mean that `PaymentConfiguration` has to be instantiated first?

Reply:
Also `PaymentConfiguration` will always grab `applicationContext` so it can really be instantiated with any context.

Reply:
I am removing this to keep it consistent with ios.  There is more than just this that needs to be done for PaymentSheet (see guide)